### PR TITLE
[feat] Display item(s) size in delete dialog

### PR DESCRIPTION
**Pull request by DoozyDoz**
### Description
- In response to feature request #2467 

### Screenshots

<p float="left">
 					
  					 
  					
				   </p>

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt [hunk 1]
```diff
@@ -1104,15 +1104,19 @@ class ViewPagerActivity : SimpleActivity(), ViewPager.OnPageChangeListener, View
     }
 
     private fun askConfirmDelete() {
-        val filename = "\"${getCurrentPath().getFilenameFromPath()}\""
+        val path = getCurrentPath()
+        val fileDirItem = FileDirItem(path, path.getFilenameFromPath(), getIsPathDirectory(path))
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt:1108]: please use the helper function File.toFileDirItem() for creating FileDirItem from path

Comment (DoozyDoz) [reply app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt]: Changes made as requested

File: app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt [hunk 2]
```diff
@@ -595,17 +595,14 @@ class DirectoryAdapter(
...
+                    val folder = path.getFilenameFromPath()
                     "\"$folder\" ($size)"
                 } else {
                     val paths = getSelectedPaths()
                     val fileDirItems = ArrayList<FileDirItem>(paths.size)
-                    paths.forEach {
-                        val fileDirItem = FileDirItem(it, it.getFilenameFromPath(), activity.getIsPathDirectory(it))
-                        fileDirItems.add(fileDirItem)
-                    }
+                    paths.forEach { fileDirItems.add(File(it).toFileDirItem(activity)) }
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt:605]: split these back into 3 lines please for better readability

File: app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt [hunk 3]
```diff
@@ -602,7 +602,10 @@ class DirectoryAdapter(
                 } else {
                     val paths = getSelectedPaths()
                     val fileDirItems = ArrayList<FileDirItem>(paths.size)
-                    paths.forEach { fileDirItems.add(File(it).toFileDirItem(activity)) }
+                    paths.forEach {
+                        val fileDirItem = FileDirItem(it, it.getFilenameFromPath(), activity.getIsPathDirectory(it))
+                        fileDirItems.add(fileDirItem)
+                    }
                     val size = fileDirItems.sumByLong { it.getProperSize(activity, countHidden = true) }.formatSize()
```
Comment (DoozyDoz) [inline app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt:609]: Changed back to three lines for readdability

Comment (tibbi) [reply app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt:609]: Thats 4... I mean that you should still use File(it).toFileDirItem(activity), just add new lines around brackets

Comment (DoozyDoz) [reply app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt:609]: Am sorry for the mistake I changed back to File(it).toFileDirItem(activity) in the last commit

File: app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/MediaAdapter.kt [hunk 4]
```diff
@@ -462,15 +463,24 @@ class MediaAdapter(
...
         } else {
-            resources.getQuantityString(R.plurals.delete_items, itemsCnt, itemsCnt)
+            val paths = getSelectedPaths()
+            val fileDirItems = ArrayList<FileDirItem>(paths.size)
+            paths.forEach {
+                val fileDirItem = File(it).toFileDirItem(activity)
+                fileDirItems.add(fileDirItem)
+            }
+            val size = fileDirItems.sumByLong { it.getProperSize(activity, countHidden = true) }.formatSize()
+            "${resources.getQuantityString(R.plurals.delete_items, itemsCnt, itemsCnt)} ($size)"
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/MediaAdapter.kt:478]: move this resource... string out to a local variable above please, this is way too much stuff within brackets

Comment (DoozyDoz) [reply app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/MediaAdapter.kt]: Created a variable for it ("deleteItemsString")

**Comment by tibbi** [comment]
looks good, but remove it from folder, from DirectoryAdapter please, it can be way too slow

**Comment by DoozyDoz** [comment]
@tibbi, I have removed the functionality for folder view

**Comment by tibbi** [comment]
ye, looks good now, thanks

**Pull request merged**